data_preparation.R

```diff
@@ -310,10 +311,23 @@ data_trend <- comb_all(data_trend_op,
   # This will be used as the x-axis in a shiny plot
   mutate(quarter_date_last = as.yearmon(quarter_date,
                                         "%d/%m/%Y")) %>%
+  # removing National Facility All Outpatient Appointments
+  filter(loc_code != "SB0801") %>%
+  # distinguishing between GJNH and NHS Louisa Jordan
+  mutate(hb_name = case_when(hb_name == "National Facility" ~ loc_name,
+                             TRUE ~ hb_name)) %>%
   
   # Arrange by date for later plotting
   arrange(quarter_date_last)
 
+### recoding GJNH and Louisa Jordan to NHS Boards
+nf_data_trend = data_trend %>% 
+  filter(hb_code == "SB0801") %>%
+  mutate(geo_type = "NHS Board of treatment")
+
+### adding GJNH and LJ as boards
+data_trend = full_join(data_trend, nf_data_trend)
```

Comment:
Does the full join create duplicate variables like hb_name.x etc? Would bind_rows be better?

Reply:
I did check and it doesn't duplicate variables - if I'd specified only some variables to join by, it would duplicate the ones I'd excluded but it joins on everything.

Reply:
oh great! :)